components/Schema.js

```diff
@@ -103,12 +103,20 @@ function SchemaProp({ prop, propName, required = false, path = '', description =
 
 function SchemaPropRow({ prop, propName, required = false, path = '', description = '', isCircular = false }) {
   const acceptedValues = prop.enum() && prop.enum().length ? prop.enum().join(', ') : '_Any_';
-  const types = `${prop.anyOf() ? `anyOf` : ''}${prop.allOf() ? `allOf` : ''}${prop.oneOf() ? `oneOf` : ''}${prop.items() && !Array.isArray(prop.items()) && prop.items().type() ? `${prop.items().type()}` : ''}`;
-  description = `${description || prop.description() || ''}${isCircular ? ' **[CIRCULAR]**': ''}`
+  let type_array = [
+      prop.type() || '',
+      prop.anyOf() ? `anyOf` : '',
+      prop.allOf() ? `allOf` : '',
+      prop.oneOf() ? `oneOf` : '',
+      prop.items() && !Array.isArray(prop.items()) && prop.items().type() ? prop.items().type() : ''
+  ];
+  const types = type_array.filter(type => type != '').join(', ');
+
+  description = `${description || prop.description() || ''}${isCircular ? ' **[CIRCULAR]**': ''}`.replace(new RegExp('\r?\n','g'), '<br />');
```

Comment:
By using the HTML tags we can have again problems described in this https://github.com/asyncapi/markdown-template/issues/33 issue.

Reply:
Hm, I don't know a better solution.
Should we drop newlines at all?

Reply:
Yeah, at the moment we can drop the newlines at all, because HTML tags are only supported by Flavour Markdown used in Github. For example, In JIRA we must use the `//` to make new lines, so yeah, dropping will be the best option here. So you can change the code to:

```suggestion
  description = `${description || prop.description() || ''}${isCircular ? ' **[CIRCULAR]**': ''}`.replace(new RegExp('\r?\n','g'), ' ');
```

Reply:
Ok, I do `.replace(new RegExp('\S*\r?\n','g'), ' ');` to catch trailing spaces.